lib/tasks/cucumber.rake

```diff
@@ -41,10 +53,12 @@ begin
       ::CodeStatistics::TEST_TYPES << "Cucumber features" if File.exist?('features')
     end
   end
-  desc 'Alias for cucumber:ok'
-  task :cucumber => 'cucumber:ok'
 
+  desc 'Alias for cucumber:ok'
   task :default => :cucumber
+  task :cucumber => 'cucumber:aa'
```

Comment:
Yes, precisely that.  Except I would run them in alphabetical order like cucumber does but that's extremely minor which order they run in.